lib/polyfills.js

```diff
@@ -5,5 +5,11 @@
 
 // internal-ip@2.x uses [].includes
 if (!Array.prototype.includes) {
-  Array.prototype.includes = require('array-includes');
+  const includes = require('array-includes');
+  Object.defineProperty(Array.prototype, 'includes', {
```

Comment:
would probably be better to use `includes.shim();` here, no? https://github.com/ljharb/array-includes/blob/master/shim.js